New Entry and Terms: Java `StringBuilder` (#870)
Codecademy/docs: content/java/concepts/stringbuilder/terms/replace/replace.md

---
Title: '.replace()'

> Description: 'Replaces a substring in a StringBuilder with the specified String.'

Reviewer: ```suggestion
Description: 'Switches a substring in a StringBuilder with a specified String and returns the modified object.'
```

Subjects:
  - 'Computer Science'
Tags:
  - 'Characters'
  - 'Strings'
  - 'Methods'
CatalogContent:
  - 'learn-java'
  - 'paths/computer-science'
---

> The **`.replace()`** method replaces a substring in a `StringBuilder` with the specified [`String`](https://www.codecademy.com/resources/docs/java/strings). It returns the `StringBuilder`.

Reviewer: ```suggestion
The **`.replace()`** method switches a substring in a `StringBuilder` with a specified [`String`](https://www.codecademy.com/resources/docs/java/strings) and returns the modified object.
```
